### ISSUE TITLE
Modifiers in Til Death score tab are displayed wrongly

### ISSUE THREAD

**Issue by kangalioo** [etternagame/etterna#753]
**Describe the bug**
Score modifiers are displayed wrongly in Til Death score display

**To Reproduce**
1. Find a score with No Mines or Fail End Of Song on
2. Go to Scores tab
2. See error in "Mod:" row

**Expected behavior**
The game shouldn't insert commas inbetween all words.

**Screenshots**
![Screenshot_20200503_120506](https://user-images.githubusercontent.com/21220820/80911426-9a9dba80-8d36-11ea-99c6-175a492350ad.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - Version: 0.69 76d77af1a7
 - 64bit

**Comment by poco0317** [comment]
The root of the issue lies here https://github.com/etternagame/etterna/blob/ad135f144aff93b7979e68200574f0c7a1b6cbc2/Themes/_fallback/Scripts/00%20Utility.lua#L314-L320

However, an alternative fix can be created.
One idea would be to rename Fail End Of Song to EndOfSong like it used to be. This is a translation issue.

**Comment by kangalioo** [comment]
Can't you just change the regular expression from `[^,%s]+` to `[^,]+`? That way space characters would not be counted as a separator.

**Comment by poco0317** [comment]
Upon further review I can't reproduce this.
![image](https://user-images.githubusercontent.com/2531164/81763702-7314c380-9495-11ea-9bfe-ea746e0a5203.png)

**Comment by kangalioo** [comment]
Hm I don't know why it doesn't reproduce for you. Maybe you've got a more recent version in which someone already fixed the issue (maybe mina fixed it but with an obscure commit description?).

Anyway, this was pretty easy to fix locally by replacing
```lua
for mod in string.gmatch(source, "[^,%s]+") do
```
with
```lua
for mod in string.gmatch(source, "[^,]+") do
	mod = mod:match("^%s*(.-)%s*$") -- trim whitespace from beginning and end
```
What this does is: instead of blindly splitting on every whitespace or comma, this only splits on commas and removes superfluous whitespace afterwards

**Comment by MinaciousGrace** [comment]
i haven't touched anything in the same solar system as this stuff, so it wasn't me

**Comment by poco0317** [comment]
View the same score on a previous version to verify this is a new 0.67+ issue.

**Comment by poco0317** [comment]
fixed in 4ccb3559edeb57dcd324cd5296f2411c9e97c52a
pending closure at merge
